### DISCUSSION TITLE
How can I get automatic connection working in Windows?

### DISCUSSION THREAD

**Discussion by elp3dr0** [Martchus/syncthingtray#301] [Q&A] [answered]
How do I get SyncthingTray to connect automatically at login when using an external syncthing binary? The external syncthing binary starts up by itself - so does not need to be started by syncthing tray. SynchthingTray starts when the user logs on, but it never connects when the user logs on to the windows machine. As soon as I click the 'connect' button in syncthing tray, it connects fine.

Settings are as follows:

![Screenshot 2024-12-10 095913](https://github.com/user-attachments/assets/2195403f-3f4f-42b0-abbc-0cb8aa44f61d)
![Screenshot 2024-12-10 100021](https://github.com/user-attachments/assets/d1127877-d71f-41e4-b565-7a47edd460a3)
![Screenshot 2024-12-10 100101](https://github.com/user-attachments/assets/ca27f8da-0ea4-47a0-a64d-951a29bcfc89)

Windows 11 Pro 24H2
Syncthingtray 1.6.4
Syncthing 1.28.1

I'm starting

**Comment by Martchus** [comment, marked as answer]
Uncheck "Consider process status for notifications and re-connect attempts …". Since you're not starting Syncthing via Syncthing Tray the state of Syncthing Tray's launcher cannot be used to improve handling of notifications and re-connect attempts. With this setting enabled, Syncthing Tray thinks Syncthing is not running (because it isn't started via its built-in launcher in your case) and therefore avoids re-connect attempts.

**Reply by elp3dr0** [reply]
That works! Thanks
